src/plugins/dashboard/public/dashboard_container/embeddable/dashboard_container_factory.tsx

```diff
@@ -14,13 +14,13 @@ import { EmbeddablePersistableStateService } from '@kbn/embeddable-plugin/common
 import {
   Container,
   ContainerOutput,
-  EmbeddableAppContext,
   EmbeddableFactory,
   EmbeddableFactoryDefinition,
   EmbeddablePackageState,
   ErrorEmbeddable,
 } from '@kbn/embeddable-plugin/public';
 import { IKbnUrlStateStorage } from '@kbn/kibana-utils-plugin/public';
+import { EmbeddableAppContext } from '@kbn/presentation-publishing';
```

Comment:
Looks like Canvas also uses this type in `x-pack/plugins/canvas/canvas_plugin_src/renderers/embeddable/embeddable.tsx`.

Should we import from the `presentation-publishing` package there as well?

Reply:
Woops - good catch! Thought I got them all. Done in https://github.com/elastic/kibana/pull/181628/commits/8a74fe6986987bf9e82c5d4e23c6e06cc79b45c9